### ISSUE TITLE
Ability to crop and uncrop the buffer

### ISSUE THREAD

**Issue by jerryepratt** [ihmcrobotics/simulation-construction-set-2#169]
There are times when you want to zoom in on some data. One way is to crop the buffer. But then you lose that data.

Feature request: 

Some sort of mode where you only see the data between the in and out point, but then can switch to see all the data (so that you can reset the in and out points too).

One option is to all for "uncropping" a buffer.

Another option is to add a checkbox in the DataBuffer menu,  where when the check box is checked, you don't see any of the data outside of the in point and out point. Then when you uncheck it, you see that data again.

**Comment by calvertdw** [comment]
I had this issue too and then I saw Sylvain use the plus and minus magnifying glass buttons to zoom in and out, and I felt dumb for a second. Do those buttons solve your issue?
![image](https://github.com/ihmcrobotics/simulation-construction-set-2/assets/1161586/c4f0b7f9-4125-4d41-bb07-483b2cb5ca1c)

**Comment by jerryepratt** [comment]
> I had this issue too and then I saw Sylvain use the plus and minus magnifying glass buttons to zoom in and out, and I felt dumb for a second. Do those buttons solve your issue? ![image](https://private-user-images.githubusercontent.com/1161586/310310259-c4f0b7f9-4125-4d41-bb07-483b2cb5ca1c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDk2ODE4MzQsIm5iZiI6MTcwOTY4MTUzNCwicGF0aCI6Ii8xMTYxNTg2LzMxMDMxMDI1OS1jNGYwYjdmOS00MTI1LTRkNDEtYmIwNy00ODNiMmNiNWNhMWMucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDMwNSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDAzMDVUMjMzMjE0WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NDdkYWIyNmI5MjQyZDc0ODVlZmUyOGFmMzcwN2RiYmYzMDVjNmNhODliMjdjMGE1MmI0ZTg4NmFlNjE5MDJiYyZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.YY2ALja6IkgnwguSTO2EyMUnyg594AN5u57wznLVIKU)

Nope. There are additional reasons to entirely remove the other data from view. Some of the reasons include having the start of the data align with the left side of the screen, and the right with the right side without having to do a bunch of drags and clicks. Another one is when you are dragging the timeline bar, if you go off the right side of the screen and out of the data window that you care about, it takes a bit of effort to get back to where you were. By having the other data removed from the view, your cursor will stop once it hits the edges of the data window, rather than pulling in data you don't care about.

Being able to switch between these modes really quickly will be helpful. Something like a checkbox on the button list, maybe near the +/- buttons would be good.
